students/ThanhNhan/lesson03/list_lab.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+#Series 1
+fruit =['Apples', 'Pears', 'Oranges', 'Peaches']
+print (fruit) #display a list of fruit
+response = input("Add another fruit > ")
+print (response) #display food to add
+fruit.append(response)
+print(list(enumerate(fruit, start=1)))
+#print (fruit)
+num_disp = input("Enter Number and fruit display back to user >")
+print (fruit[int(num_disp) - 1])
+add_a_fd = input("Add another fruit from the beginning using >") 
+fruit= [add_a_fd] +  fruit #add fruit using +
+print (fruit)
+add_a_finsert = input("Add another fruit from the beginning using >") 
+fruit.insert(0, add_a_finsert) #add fruit using insert
+print (fruit)
+print ("Display all the fruit that begin with P:")
+for i in range(len(fruit)):
+	if(fruit[i][0] == "P"):
```

Comment:
You'll want to keep all indentions in Python to 4 spaces.  Also, good idea to use lower() here to catch any lowercase input from users.